docs/csharp/whats-new/csharp-version-history.md

```diff
@@ -28,11 +41,18 @@ Now things start to get interesting. Let's take a look at some major features of
 - [Iterators](../programming-guide/concepts/iterators.md)
 - [Covariance and contravariance](../programming-guide/concepts/covariance-contravariance/index.md)
 
-While C# may have started as a pretty generic Object-Oriented (OO) language, C# version 2.0 changed that in a hurry. Once they had their feet under them, they went after some serious developer pain points. And they went after them in a big way.
+Other C# 2.0 features added capabilities to existing features:
+
+- Getter/setter separate accessibility
+- Method group conversions (delegates)
+- Static classes
+- Delegate inference
+
+While C# may have started as a generic Object-Oriented (OO) language, C# version 2.0 changed that in a hurry. Once they had their feet under them, they went after some serious developer pain points. And they went after them in a significant way.
 
-With generics, you have types and methods that can operate on an arbitrary type while still retaining type safety. So, for instance, having a <xref:System.Collections.Generic.List%601> lets you have `List<string>` or `List<int>` and perform type safe operations on those strings or integers while you iterate through them. This is better than creating `ListInt` inheritors or casting from `Object` for every operation.
+With generics, you have types and methods that can operate on an arbitrary type while still retaining type safety. So, for instance, having a <xref:System.Collections.Generic.List%601> lets you have `List<string>` or `List<int>` and perform type safe operations on those strings or integers while you iterate through them. Using generics is better than creating `ListInt` inheritors or casting from `Object` for every operation.
```

Comment:
nit: you have types and methods that can... --> types and methods can
nit: So, for instance, --> For instance,
type safe --> type-safe
what are ListInt inheritors?